### PR TITLE
[Mikhail Ilichev] Tasks 91-100

### PR DESCRIPTION
**Pull request by miffn3**

### CODE REVIEW

File: tasks/91_make-a-person/make-a-person.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  this.getFullName = function() {
+    return firstName + ' ' + lastName;
+  };
+  this.setFirstName = function(firstNameNew) {
+    firstName = firstNameNew;
+  };
+  this.setLastName = function(lastNameNew) {
+    lastName = lastNameNew;
+  };
+  this.setFullName= function(fullName) {
```
Comment (KalachevDev) [inline tasks/91_make-a-person/make-a-person.js]: space?

File: tasks/91_make-a-person/make-a-person.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ *
+ * @param {String} fullName - first name and last name of person.
+ */
+export function Person(fullName) {
+  const name = fullName.split(' ');
+  let firstName = name[0];
+  let lastName = name[1];
+  this.getFirstName = function() {
+    return firstName;
+  };
```
Comment (KalachevDev) [inline tasks/91_make-a-person/make-a-person.js:11]: spaces?

File: tasks/91_make-a-person/make-a-person.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+/**
+ * Person - Class that contains information about person' full name.
+ *
+ * @param {String} fullName - first name and last name of person.
+ */
+export function Person(fullName) {
+  const name = fullName.split(' ');
```
Comment (KalachevDev) [inline tasks/91_make-a-person/make-a-person.js]: let [firstName, lastName] = fullName.split(' ');

Comment (KalachevDev) [reply tasks/91_make-a-person/make-a-person.js]: Moreover, do we really need to introduce variables here? Why can't we just do as `let splittedName = fullName.split(' ')` and operate with this variable?

File: tasks/100_traverse-tree/traverse-tree.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+ * traverseTree - Perform recursive traverse of tree.
+ *
+ * @param {Object} node - current node of tree.
+ * @param {Function} adder - executes on each node.
+ */
+export function traverseTree(node, adder) {
+  for (let i = 0; i < node.children.length; i++) {
+    traverseTree(node.children[i], adder);
+  }
+  adder(node);
```
Comment (KalachevDev) [inline tasks/100_traverse-tree/traverse-tree.js:11]: Why don't you call `adder` on node before you start loop?

Comment (miffn3) [reply tasks/100_traverse-tree/traverse-tree.js:11]: @KalachevDev we don't care about traverse order because in tests result array will be sorted

**Review by alexkond — APPROVED** [review]

**Review by savchukm — APPROVED** [review]

**Pull request merged**